mailpoet/assets/js/src/newsletters/campaign_stats/page.tsx

```diff
@@ -65,7 +65,7 @@ function CampaignStatsPage({ match, history, location }: Props) {
         loading: false,
         item: response.data,
       });
-    }).fail((response) => {
+    }).fail((response:{errors:{message:string}[]}) => {
```

Comment:
I think you could define an alias for this type as it's used in multiple places.

Reply:
Addressed in 12eeced71